lib/instances/BindableFunction.lua

```diff
@@ -0,0 +1,52 @@
+local BaseInstance = import("./BaseInstance")
+local InstanceProperty = import("../InstanceProperty")
+
+local BindableFunction = BaseInstance:extend("BindableFunction", {
+	creatable = true,
+})
+
+-- This isn't type checked because Roblox doesn't.
+-- Instead, you just get a "attempt to call a [whatever] value" error when you :Invoke()

```

Comment:
I think this is a case where it might be okay to be stricter than Roblox, since allowing non-function values here is nonsense.

Reply:
Is there an easy way to combine two InstanceProperties? I want to make it writeOnly and typed.